tests/test_validation.py

```diff
@@ -401,19 +404,21 @@ def test_raster_projected_in_m(self):
         raster = None
 
         for unit in ('m', 'meter', 'metre', 'meters', 'metres'):
+            pint_unit = utils.u.Unit(unit)
             error_msg = validation.check_raster(
-                filepath, projected=True, projection_units=unit)
+                filepath, projected=True, projection_units=pint_unit)
             self.assertEqual(error_msg, None)
 
         # Check error message when we validate that the raster should be
         # projected in feet.
         error_msg = validation.check_raster(
-            filepath, projected=True, projection_units='feet')
-        self.assertTrue('projected in feet' in error_msg)
+            filepath, projected=True, projection_units=utils.u.foot)
+        expected_msg = "Layer must be projected in this unit: 'foot'"
+        self.assertEqual(error_msg, expected_msg)
```

Comment:
This added assertion seems like basically the same case that's tested in the next test: `test_raster_incorrect_units`. Also it doesn't really match this test's description. If it really is redundant with the next test, maybe we can remove this one?

Reply:
I think this one was there from before I added `pint`, validation would specifically check for the meter aliases `m, meter, meters, metre, metres`. Agree this can be removed!